scheduler/scheduler.go

```diff
@@ -116,18 +116,11 @@ func (s *distroScheduler) scheduleDistro(distroID string, runnableTasksForDistro
 	})
 
 	// persist the queue of tasks and its associated distroQueueInfo
-	persistTime := time.Now() // Get time before persisting tasks so that scheduled time is never after start time.
```

Comment:
I added this recently https://jira.mongodb.org/browse/EVG-6422. Why are you removing it?

Reply:
because PersistTaskQueue does this now, this call is duplicative.